### ISSUE TITLE
Align change requests permissions between projects and environments

### ISSUE THREAD

**Issue by kyle-ssg** [SolidStateGroup/bullet-train-api#4869]
### Is your feature request related to a problem? Please describe.

https://github.com/Flagsmith/flagsmith/issues/4044 raises a disparity between project level and environment level change requests.

With it we have the following permissions which have differences in permission behaviour and terminology

### **Project**

Manage segments
Ability to manage segments in the given project.

Manage project level change requests
Ability to publish, create and delete change requests.

Approve project level change requests
Ability to approve project level change requests.

###  **Environment** 

Update feature state
Update the state or value for a given feature state.  //or publish change requests

Create change request
Permission to create change requests in the given environment.

Approve change request
Permission to approve change requests in the given environment.

### Describe the solution you'd like.

We should have a consistent approach to these permissions:

Option 1: 
- Create Change requests: Create change requests
- Approve Change requests: Approve change requests
- Manage segments/Update Feature state - Used to either publish change requests or save the item if change requests are disabled. We should include this in the description

Option 2: 
- Manage Change requests: Create and publish
- Approve Change requests: Approve change requests
- Manage X - Only visible if change requests are disabled since they are superseded by change requests
  

- We should update the mange segment permission description to specify that it means create and update

### Describe alternatives you've considered

Options covered in above

**Comment by matthewelwell** [comment]
@kyle-ssg I agree with this. I think option 1 makes the most sense at this point and gives us the flexibility in the future to still extend it to split out the behaviour in CR / non-CR environments. 

@zachaysan can you scope this change out and let me know how much effort it will be to do before we release it?

**Comment by zachaysan** [comment]
@matthewelwell if we're going with option #1 I think the change is just to update the existing permission with broader language to cover the use cases that we already cover, no?

**Comment by matthewelwell** [comment]
@zachaysan not quite I don't think: 

---

>Manage project level change requests
>Ability to publish, create and delete change requests.

--- 

>Create change request
>Permission to create change requests in the given environment.

---

I think the crux of it is that these 2 should be aligned.
